services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -1128,6 +1128,26 @@ CREATE OR REPLACE PROCEDURE
   END;
 $$
 
+CREATE OR REPLACE PROCEDURE
+  add_fact_source_and_description_to_environment_fact()
+
+  BEGIN
+    IF NOT EXISTS(
+      SELECT NULL
+      FROM INFORMATION_SCHEMA.COLUMNS
+      WHERE
+        table_name = 'environment_fact'
+        AND table_schema = 'infrastructure'
+        AND column_name = 'source'
+    ) THEN
+      ALTER TABLE `environment_fact`
+      ADD `source` varchar(300) NOT NULL default ''
+      ADD `description` TEXT NULL DEFAULT ''
+      ALTER TABLE `environment_fact`
```

Comment:
@timclifford 
there is a too much `ALTER TABLE` in there, plus also a semicolon missing

Reply:
@Schnitzel added a fix